docs/src/main/asciidoc/neo4j.adoc

```diff
@@ -4,7 +4,7 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 ////
 = Quarkus - Neo4j
-:neo4j_version: 3.5.6
+:neo4j_version: 3.5.13
```

Comment:
Hmmm, well, I wasn't entirely clear but my main concern was this one :).

Reply:
Oh sorry, I didn't get that :/